source/user-portal/assets/script.js

```diff
@@ -1,9 +1,7 @@
 // Script for adding recipe card
 
-// This would be the array from Local Storage
 const recipes = [
-
-
+    'https://introweb.tech/assets/json/birthdayCake.json'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8436327)
```suggestion
    "https://introweb.tech/assets/json/birthdayCake.json"
```